RegularExpressionAndExceptionHandling/RegularExpressionAndExceptionHandling/ValidPhoneNumber.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+
+namespace RegularExpressionAndExceptionHandling
+{
+    /// <summary>
+    /// ValidPhoneNumber class uses a regular expression to validate all valid forms of United States phone numbers.
+    /// It passes in a string, phoneNumber, to validate against a pattern. 
+    /// </summary>
+    public static class ValidPhoneNumber
+    {
+        public static bool PhoneNumber(string phoneNumber)
```

Comment:
This method should have an XML comment (C1) (-0.5). Also, method names should typically indicate what they do - in this case, it would be good to name the method something like IsValid or IsValidPhoneNumber or ValidatePhoneNumber.